src/redismodule.h

```diff
@@ -69,6 +69,23 @@
 #define REDISMODULE_HASH_CFIELDS    (1<<2)
 #define REDISMODULE_HASH_EXISTS     (1<<3)
 
+/* StreamID type. */
+typedef struct RedisModuleStreamID {
+    uint64_t ms;
+    uint64_t seq;
+} RedisModuleStreamID;
+
+/* StreamID MIN and MAX to be used like "-" and "+" in range queries. */
+#define REDISMODULE_STREAMID_MIN \
+    ((struct RedisModuleStreamID){0, 0})
+#define REDISMODULE_STREAMID_MAX \
+    ((struct RedisModuleStreamID){UINT64_MAX, UINT64_MAX})
```

Comment:
These are not yet documented (documentation is generated from the comments in the C file.
i guess you're not using them, and just added them as a preparation for your next PR (in which case let's trim them from here), or are you planning to extend this PR with more stream APIs?

Reply:
That's right, they're not used yet. I can remove them and add them again in another PR where they're used. I think it's good to divide the stream API into multiple PRs.

Are the docs generated from the comments? I assumed they're just manually copied into the redis-docs repo. Please clarify so I can do it properly.

Reply:
the docs are generated by `src/modules/gendoc.rb` (parsing module.c and looking for comments above RM_ apis.